open_spiel/python/mfg/algorithms/best_response_value.py

```diff
@@ -75,10 +75,18 @@ def eval_state(self, state):
       return self._state_value[state_str]
     elif state.current_player() == pyspiel.PlayerId.MEAN_FIELD:
       dist_to_register = state.distribution_support()
+      def get_state_probability(str_state):
+        try:
+          return self._distribution.value_str(str_state)
+        except ValueError:
```

Comment:
Also here, don't use the exception mechanism for something that can happen under normal conditions.
I would modify the Distribution API to allow for an argument with a default value when the state is not in the distrib, so that you could call value_str(str_state, 0.)

Reply:
Thanks